src/hero.css

```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
This part of the code is not required as it is already covered in [base.css](https://github.com/ComputerSocietyVITC/docHub/blob/master/css/base.css)